ops/ansible/roles/legislation_explorer/defaults/main.yml

```
@@ -0,0 +1,33 @@
+legislation_explorer_api_url: http://openfisca-api.example
+legislation_explorer_base_url_path: /legislation
+legislation_explorer_branch_name: master
+legislation_explorer_changelog_url: https://github.com/openfisca/openfisca-france/blob/master/CHANGELOG.md
+legislation_explorer_host: 127.0.0.1
+legislation_explorer_http_port: 8000
```

Could we set the default to something that would not collide with the default API port? 🙂 
```suggestion
legislation_explorer_http_port: 8001
```